src/styles/blocks/catalog/card.scss

```diff
@@ -0,0 +1,100 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  row-gap: 46px;
+  column-gap: 48px;
+  grid-auto-flow: row;
+  justify-content: center;
+  align-content: center;
+
+  justify-items: center;
+  align-items: center;
+  margin-block: 50px 72px;
```

Comment:
Checklist 8

Reply:
I know, but I don't understand how to add mix here(